typescript/index.js

```diff
@@ -15,10 +15,12 @@ const genTypes = pipe(
   join('\n\n')
 )
 
-function typescript (swaggerJson) {
-  const { components: { schemas } } = swaggerJson
-
-  const file = genTypes(schemas)
+function typescript(swaggerJson) {
+  if (!swaggerJson.components || (swaggerJson.components && !swaggerJson.components.schemas)) {
+    return console.log('ERROR: schemas property is not available')
+  }
+  
+  const file = genTypes(swaggerJson.components.schemas)
```

Comment:
I would have to verify but I would guess that the swagger spec does not mention `swagger.components.schemas` as a required property for the validity of the swagger document.

The safeguard is a good addition, but I don't think we need an error for missing schemas.

Let's just make sure we don't call genTypes if `schemas` doesn't exist, but not consider it an error.

Also, use [optional chaning](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here for the existence checks
```javascript
swaggerJson?.components?.schemas
```

Or default destructuring arguments
```javascript
const { components: { schemas }  } = swaggerJson
```

Reply:
the problem came from my openapi version. I used 2.0. I updated the description of this PR

Reply:
solved